exodus/restful_api/views.py

```diff
@@ -31,10 +31,16 @@ def get_report_infos(request, r_id):
         except Report.DoesNotExist:
             raise Http404('No report found')
 
+        certificate = None
+        if hasattr(report.application, 'apk'):
+            certificates = Certificate.objects.filter(apk=report.application.apk)
+            certificate = certificates.first()
```

Comment:
Did you remove the check here on purpose ?

Reply:
if there is no match ; certificates returns en empty queryset ; calling first() on that returns None ; which is the default [value choosen for certificate variable](https://github.com/Exodus-Privacy/exodus/pull/314/files/66eaf722a4d0814577c639b77097dde314841855#diff-ac511c37c85d0371b7bb572ac0f59086R34)

Reply:
Perfect, thanks!